### PR TITLE
Fork accelerator builds to use the EL + kickstart build

### PR DESCRIPTION
**Pull request by jjerger**
More native than the derivative images we were building. We will also need to modify the linux build pipeline to use these.

### CODE REVIEW

File: daisy_workflows/build-publish/enterprise_linux/rocky_linux_8_optimized_gcp_with_nvidia_550_ks.publish.json
Comment (a-crate) [inline daisy_workflows/build-publish/enterprise_linux/rocky_linux_8_optimized_gcp_with_nvidia_550_ks.publish.json:1]: Drop the `ks` suffix, you can delete the rocky 8 preview publish configurations

File: daisy_workflows/image_build/enterprise_linux/rocky_linux_8_optimized_gcp_with_nvidia_550_ks.wf.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+  "Name": "build-rocky-8-optimized-gcp-2204-with-nvidia-550-ks",
+  "Vars": {
+    "nvidia_driver_version": {
```
Comment (a-crate) [inline daisy_workflows/image_build/enterprise_linux/rocky_linux_8_optimized_gcp_with_nvidia_550_ks.wf.json:4]: This is unnecessary now

**Review by a-crate — COMMENTED** [review]
Switch from `550` to `latest` here - Rocky doesn't have any metapackages for specific versions right now so we are just getting whatever the newest driver release is.

**Comment by jjerger** [comment]
Adjusted all files from 550 -> Latest

File: daisy_workflows/build-publish/enterprise_linux/rocky_linux_8_optimized_gcp_with_nvidia_latest.publish.json [hunk 2]
```diff
@@ -33,9 +33,9 @@
   {{$time := trimPrefix .publish_version "v"}}
   "Images": [
     {
-      "Prefix": "rocky-linux-8-optimized-gcp-with-nvidia-550",
-      "Family": "rocky-linux-8-optimized-gcp-with-nvidia-550",
-      "Description": "Rocky Linux 8 optimized for GCP with Nvidia 550 built on {{$time}}",
+      "Prefix": "rocky-linux-8-optimized-gcp-nvidia-latest",
+      "Family": "rocky-linux-8-optimized-gcp-nvidia-latest",
+      "Description": "Rocky Linux 8 optimized for GCP with latest LTS Nvidia driver built on {{$time}}",
```
Comment (a-crate) [inline daisy_workflows/build-publish/enterprise_linux/rocky_linux_8_optimized_gcp_with_nvidia_latest.publish.json:38]: Not necessarily latest LTS actually, just latest

File: daisy_workflows/build-publish/enterprise_linux/rocky_linux_8_optimized_gcp_with_nvidia_latest.wf.json [hunk 3]
```diff
@@ -9,8 +9,8 @@
       "Description": "Build datestamp used to version the image."
     },
     "google_cloud_repo": {
-      "Value": "DEPRECATED",
-      "Description": "DEPRECATED. Included to ease transition of our pipelines, but not used."
+      "Value": "stable",
+      "Description": "The Google Cloud Repo branch to use."
```
Comment (a-crate) [inline daisy_workflows/build-publish/enterprise_linux/rocky_linux_8_optimized_gcp_with_nvidia_latest.wf.json:13]: Shouldn't this still be DEPRECATED?

File: daisy_workflows/image_build/enterprise_linux/kickstart/rocky_linux_8_optimized_gcp_accelerator.cfg [hunk 4]
```diff
@@ -0,0 +1,261 @@
...
+
+# Install GCE guest packages.
+dnf install -y google-compute-engine google-osconfig-agent gce-disk-expand
+
+# Install the Cloud SDK package.
+dnf install -y google-cloud-cli
+
+# Install Accelerator components: nvidia and mellanox drivers
+dnf config-manager --add-repo https://developer.download.nvidia.com/compute/cuda/repos/rhel8/x86_64/cuda-rhel8.repo
+dnf install -y gcc make kernel-devel kernel
```
Comment (a-crate) [inline daisy_workflows/image_build/enterprise_linux/kickstart/rocky_linux_8_optimized_gcp_accelerator.cfg:188]: Drop lines 188-200 for now - we will just include the drivers when they are available from CIQ

File: daisy_workflows/build-publish/enterprise_linux/rocky_linux_8_optimized_gcp_with_nvidia_latest.publish.json (hunk 2 above)
Comment (jjerger) [reply daisy_workflows/build-publish/enterprise_linux/rocky_linux_8_optimized_gcp_with_nvidia_latest.publish.json]: Fixed

File: daisy_workflows/build-publish/enterprise_linux/rocky_linux_8_optimized_gcp_with_nvidia_latest.wf.json (hunk 3 above)
Comment (jjerger) [reply daisy_workflows/build-publish/enterprise_linux/rocky_linux_8_optimized_gcp_with_nvidia_latest.wf.json]: Set this back to DEPRECATED, although the other rocky EL builds i looked at still show Stable. Might be that the EL builds needed to be updated.

**Review by a-crate — APPROVED** [review]

**Pull request merged**